app/assets/javascripts/application.js

```diff
@@ -21,7 +21,6 @@
 //= require selectize
 //= require dropzone
 //= require liquid/syntax-highlighting
-//= require typeahead.jquery
```

Comment:
@osahyoun how do these get included now? Or is it still just sprockets but this was unused?